build.py

```diff
@@ -641,6 +664,22 @@ def build(self):
 
 Project.add(Project_glib_openssl())
 
+class Project_graphene(GitRepo, Meson):
+    def __init__(self):
+        Project.__init__(self,
+            'graphene',
+            repo_url = 'https://github.com/ebassi/graphene',
+            fetch_submodules = False,
+            tag = None,
+            dependencies = ['ninja', 'meson', 'pkg-config', 'glib'],
+            )
+
+    def build(self):
+        Meson.build(self)

```

Comment:
I'm looking at it just now and it's surely ok to use super(..) with one base class but Project_Graphene inherit from two classes (GitRepo & Meson) and in this case I yhink is better be explicit.

In build.py the super is used only in Gtk2/Gtk3 to call the build() of the single class Project_gtk_base, in the other cases (say __init__(self, ...)) the call is explicit to the correct fun (Project.__init__(self, ...)